appserver/tests/payara-samples/pom.xml

```diff
@@ -566,6 +566,11 @@
             <artifactId>jakarta.ejb-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>2.3.3</version>
+        </dependency>
```

Comment:
I still think this is wrong. 
If we have to pull this in as a compile scope dependency then something is asking for javax classes rather than jakarta and needs updating.

Reply:
Yes, this solution was a workaround to run the test. The root problem is payara-container-common that depends on jersey-media-json-jackson. The last one has this statement in its pom.xml:
```
<exclusions>
<exclusion>
<groupId>jakarta.xml.bind</groupId>
<artifactId>jakarta.xml.bind-api</artifactId>
</exclusion>
<exclusion>
<groupId>jakarta.activation</groupId>
<artifactId>jakarta.activation-api</artifactId>
</exclusion>
</exclusions>
```
and
```
<dependency>
<!-- The last JAX-B API that has maven coordinates which do not collide with Jakarta coordinates -->
<!-- Do not update -->
<groupId>javax.xml.bind</groupId>
<artifactId>jaxb-api</artifactId>
<version>2.3.1</version>
<scope>test</scope>
</dependency>
```
I checked the newest version of this dependency and it also has these statements.

Reply:
The newest version of the 3.0.x tags, 3.0.4, has a commit conveniently titled [Let Jackson use JAXB3](https://github.com/eclipse-ee4j/jersey/commit/4aa8007bc4d85b27acf4e6a0485ecfc905c707d7) - they changed the scope of the javax dependency to test scope and brought back in the jakarta dependency.

So I think the proper fix is to reapply our patches (if still applicable) to Jersey 3.0.4 to make 3.0.4.payara-p1 and have Arquillian/Payara use that.